File: sdkwork-app-sdk-csharp/Models/BlockCheckVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class BlockCheckVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public bool? IsBlocked { get; set; }
        public string? BlockTime { get; set; }
        public string? BlockReason { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/block_check_vo.go
package types

// 拉黑检查响应
type BlockCheckVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	IsBlocked bool `json:"isBlocked"`
	BlockTime string `json:"blockTime"`
	BlockReason string `json:"blockReason"`
}
